### PR TITLE
ENH: add stub file of cm.py and script that generates it

### PR DESCRIPTION
**Pull request by beskep**

### CODE REVIEW

**Comment by neutrinoceros** [comment]
@beskep thanks for your PR. I must say the quality is impressive for a first-time-ever !
However there's one technical issue with it: I noticed that you pushed your changes to your `master` branch. This is a very common beginner gotcha, and we've all done it, however this cannot be changed within an existing PR, so I suggest you open a new one *from a branch*. Also note (this is project-dependent) that pull requests on CMasher should go to the `dev` branch, not the `master` branch (which is reserved for releases).

So in summary, here's a step-by-step guide on what to do
- checkout the dev branch
- start a new branch from there and check it out
- commit your changes
- re-open a new PR aimed at the `dev` branch

I'll close this one when you're done. Don't hesitate if you have questions !

**Comment by beskep** [comment]
Thank you for the instruction!
I requested another one.
Hope I've done it right this time.
